### PR TITLE
Status Icons

### PR DESCRIPTION
**Pull request by Geevies**
* Added status indicators to the top right of the screen that indicate when you're bleeding or drunk, or both.

image

Code partially ported from [CM-SS13](https://github.com/cmss13-devs/cmss13) (specifically `get_status_loc`), though a substantial chunk of it is my own.

### Asset Licenses
The following assets that **have not** been created by myself are included in this PR:

| Path | Original Author | License |
| --- | --- | --- |
| icons/mob/screen/hud_status.dmi : bleeding | [CM Dev Team](https://github.com/cmss13-devs/cmss13#readme) ([CM-SS13](https://github.com/cmss13-devs/cmss13)) | CC BY-SA 3.0 |

idk if i should also add it above, but i copied one of the drinks in drinks.dmi and slammed it into hud_status.dmi. it's from Aurora

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by Geevies** [comment]
i think a testmerge for this one would be good to make sure the code doesn't detonate itself

**Review by JohnWildkins — CHANGES_REQUESTED** [review]

**Review by JohnWildkins — CHANGES_REQUESTED** [review]
image
drunk status icon appeared empty

File: code/modules/mob/living/carbon/human/life.dm [hunk 1]
```diff
@@ -847,11 +847,17 @@
...
 
+	var/list/status_overlays = null
+
 /mob/living/carbon/human/can_update_hud()
 	if((!client && !bg) || QDELETED(src))
 		return FALSE
 	return TRUE
 
+// corresponds with the status overlay in hud_status.dmi
+#define DRUNK_STRING "string"
```
Comment (SleepyGemmy) [inline code/modules/mob/living/carbon/human/life.dm:858]: look again.

**Comment by Geevies** [comment]
i dont even know how i managed that, it's fixed now, tested in-game too to confirm

**Review by JohnWildkins — APPROVED** [review]
lol lmao

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**